composer.json

```diff
@@ -20,26 +20,27 @@
         }
     },
     "require": {
-        "php": ">=7.2",
-        "league/flysystem": "^2.1|^3.0",
-        "symfony/config": "^4.2|^5.0|^6.0",
+        "php": "^8.0",
```

Comment:
I'd prefer keeping it as >= (I prefer people being able to install it with PHP 9 easily even if I didn't have time to update the requirement, I doubt the package won't work on PHP 9.0)